packages/strapi-mongoose/lib/index.js

```diff
@@ -50,6 +51,7 @@ module.exports = function (strapi) {
             connectOptions.pass = password
           }
         }
+        connectOptions.ssl = ssl ? true : false;
```

Comment:
`connectOptions.ssl = ssl` is the same nop ?

Reply:
yes, ssl is a boolean. but this is for ssl is undefined